docs/blog/2019-02-06-government-open-data-site-with-gatsby/index.md

```diff
@@ -0,0 +1,237 @@
+---
+title: "How we're migrating a government open data site to Gatsby"
+date: 2019-02-05
+authors:
+  - Ryan Johnson
```

Comment:
We don't currently support multiple authors :/ Is there a primary author on this post we could use? Otherwise we'll have to implement that feature to ship this blog post.

Reply:
I can change it to reflect a primary author.

Reply:
We did modify the gatsby-starter-blog to support multiple authors when we shipped our blog, but I'm not quite happy with the way I'm doing it now (I describe why over here: https://brentryanjohnson.com/content-design-dry/)